crypto/cms/cms_dd.c

```diff
@@ -17,11 +17,13 @@
 
 /* CMS DigestedData Utilities */
 
-CMS_ContentInfo *cms_DigestedData_create(const EVP_MD *md)
+CMS_ContentInfo *cms_DigestedData_create(const EVP_MD *md,
+                                         OPENSSL_CTX *libctx, const char *propq)
 {
     CMS_ContentInfo *cms;
     CMS_DigestedData *dd;
-    cms = CMS_ContentInfo_new();
+
+    cms = CMS_ContentInfo_new_ex(libctx, propq);
```

Comment:
`CMS_ContentInfo_new_with_libctx`?

Reply:
Have we actually decided that that is the name we want to propagate?

Reply:
I thought we had

Reply:
Ok no problem I thought we still had a few new_ex about the place..

Reply:
> Ok no problem I thought we still had a few new_ex about the place..

Having made the decision and still having some `new_ex` about the place are two different things! I thought the general consensus was that this was the preferred name form. That doesn't mean that all existing name forms have been changed.